react/react1/week1/todo-tracker/src/App.js

```diff
@@ -0,0 +1,26 @@
+import "./App.css";
+import React from "react";
+import ToDoListComp from "./components/ToDoList";
+import toDoList from "./data/toDoList";
+import SearchComp from "./components/Search";
```

Comment:
You're importing SearchComp, but you've named the component 'Search'. 
Also, you're importing toDoList but are using ToDoListComp at line 21, I don't see how that could work :)
You should keep it consistent everywhere, both at where you define the component (and its filename) and where you import it. 
Finally, stick to the same naming convention (Pascal case) -> ToDoList instead of toDoList